src/InputManager.cpp

```diff
@@ -0,0 +1,42 @@
+#include "InputManager.h"
+
+InputManager::InputManager(GLFWwindow *w, Camera* cam)
+:window(w)
+,camera(cam)
+{
+}
+
+const void InputManager::CheckInput()
+{
+	KeyInput();
+	MouseInput();
+}
+
+const void InputManager::MouseInput()
+{
+	double xpos, ypos;
+	if(glfwGetMouseButton(window, GLFW_MOUSE_BUTTON_LEFT) == GLFW_PRESS)
+	{
+		// Get mouse position
+		glfwGetCursorPos(window, &xpos, &ypos);
+		float sensitivity = 0.005f;
```

Comment:
Prob better to keep this as member variable of InputManager as mouseSensitivity or something.

Can make it const as well.

Reply:
d